subsys/bluetooth/mesh/adv.h

```diff
@@ -75,18 +77,18 @@ int bt_mesh_adv_start(const struct bt_le_adv_param *param, int32_t duration,
 		      const struct bt_data *sd, size_t sd_len);
 
 static inline void bt_mesh_adv_send_start(uint16_t duration, int err,
-					  const struct bt_mesh_send_cb *cb,
-					  void *cb_data)
+					  struct bt_mesh_adv const *adv)
```

Comment:
```suggestion
					  struct bt_mesh_adv *adv)
```

send_end on the other hand could use a const.